charts/datadog/values.yaml

```diff
@@ -1130,6 +1130,12 @@ agents:
   #   # agent version >= 6.15
   #   # jmx_use_container_support: true
 
+  apparmor:
+    ## @param enabled - boolean - required
+    ## If true, enable apparmor enforcement, https://kubernetes.io/docs/tutorials/clusters/apparmor/
+    #
+    enabled: true
+
```

Comment:
sorry, I haven't saw it before, but could you move this new parameter inside `agents.podSecurity`

```yaml
agents:
  podSecurity:
     apparmor:
        enabled: true

```

Reply:
Yep, no problem! Fixed.